hermes/probe/create_file.go

```diff
@@ -0,0 +1,123 @@
+package probe
+ 
+import (
+   "context"
+   "crypto/sha1"
+   "fmt"
+   "io"
+   "math/rand"
+   // "os"
+   // "sync"
+ 
+   // "cloud.google.com/go/storage"
+   "github.com/googleapis/google-cloud-go-testing/storage/stiface"
+   "github.com/google/cloudprober/logger"
+   // "github.com/google/cloudprober/metrics"
+   // "github.com/google/cloudprober/probes/options"
+   // "github.com/google/cloudprober/targets/endpoint"
+   // "github.com/googleinterns/step224-2020/blob/master/hermes/hermes.go"
+)
+ 
+const (
+   begin = 1  // ID of the first HermesFile
+   end   = 50 // ID of the last HermesFile
+   maxFileSize = 1000 // maximum allowed file size in bytes
+)
+ 
+type RandomHermesFile struct {
+   ID   int64 // ID is a positive integer in the range [1,50]
+   Size int   // File size in bytes
+}
+ 
+type randomHermesFileReader struct {
+   size int // size in bytes
+   i    int // currently reading this byte
+   rand *rand.Rand
+}
+ 
+func (r *randomHermesFileReader) readDone() bool {
+   return r.i >= r.size
+}
+ 
+func (r *randomHermesFileReader) bytesLeft() int {
+   return (r.size - r.i)
+}
+ 
+func (r *randomHermesFileReader) bufferTooLong(buffer []byte) bool {
+   return len(buffer) > r.bytesLeft()
+}
+ 
+func (r *randomHermesFileReader) Read(buff []byte) (n int, err error) {
+   if r.readDone() {
+       return 0, io.EOF
+   }
+   b := buff
+   if r.bufferTooLong(b){
+       b = buff[:r.bytesLeft()]
+   }
+   n, err = r.rand.Read(b)  //n is the length  of the buffer
+   if(err != nil){
+       return  n, err// 0 as nil casn't be returned as a type int argument
```

Comment:
nit: leave two spaces before the comment.
```suggestion
       return  n, err  // 0 as nil can't be returned as a type int argument
```